sharepoint/sharepoint-ps/sharepoint-online/Set-SPOListVersionPolicy.md

```diff
@@ -0,0 +1,183 @@
+
+Sets the version policy setting on the document library.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+Set-SPOListVersionPolicy -Site https://contoso.sharepoint.com/sites/site1 -List "Documents" -EnableAutoExpirationVersionTrim $true
+```
+Example 1 sets Automatic version Storage Limits on the document library.
+
+### EXAMPLE 2
+
+```powershell
+Set-SPOListVersionPolicy -Site https://contoso.sharepoint.com/sites/site1 -List "Documents" -EnableAutoExpirationVersionTrim $false -MajorVersionLimit 500 -MajorWithMinorVersionsLimit 20 -ExpireVersionsAfterDays 30
+```
+
+Example 2 sets Manual Version Storage Limits on the document library by limiting the number of versions and the time (in days) versions are kept.
+
+### EXAMPLE 3
+
+```powershell
+Set-SPOListVersionPolicy -Site https://contoso.sharepoint.com/sites/site1 -List "Documents" -EnableAutoExpirationVersionTrim $false -MajorVersionLimit 500 -MajorWithMinorVersionsLimit 20 -ExpireVersionsAfterDays 0
+```
+
+Example 3 sets Manual Version Storage Limits on the document library by limiting the number of versions with no time limits. The new document libraries will use this version setting.
+
+## PARAMETERS
+
+### -Site
+
+Specifies the URL of the site.
+
+```yaml
+Type: SpoSitePipeBind
+Parameter Sets: (All)
+Aliases:
+Applicable: SharePoint Online
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -List
+
+The document library name or Id.
+
+```yaml
+Type: SPOListPipeBind
+Parameter Sets: (All)
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -EnableAutoExpirationVersionTrim
+Global and SharePoint admins in Microsoft 365 can set Document Library level Version History Limit settings that universally apply to new versions created.
+
+When Version History Limits are managed Automatically, SharePoint employs an algorithm behind the scenes that deletes (thins out) intermittent older versions that are least likely to be needed, while preserving sufficient high-value versions - more versions in the recent past and fewer farther back in time - in case restores are required.
+
+The valid values are:
+
+- True - Version History Limits for new versions created on the Document Library will be managed Automatically.
+- False - Version History Limits for new Versions created on the Document Library will be managed Manually by setting limits to the number of major versions (MajorVersionLimit), number of major with minor versions (MajorWithMinorVersionsLimit) and time set (ExpireVersionsAfterDays).  Review the documentation of both parameters to manage your organization's version limits Manually.  
+
+> [!NOTE]
+> When Version History Limits are managed Manually (EnableAutoExpirationVersionTrim $false), MajorVersionLimit, MajorWithMinorVersionsLimit and ExpireVersionsAfterDays are required parameters with the following acceptable values:
```

Comment:
>EnableAutoExpirationVersionTrim $false [](http://example.com/codeflow?start=52&length=38)

param names, or params with values, are usually back-tick formatted

Reply:
Updated, also other places, hopefully none is missed